locals.js

```diff
@@ -23651,11 +23653,15 @@ I18N["zh-CN"]["projects"] = { // 项目页面(含仓库项目)
             "Create project": "创建项目",
 
         // 仓库 项目页面  /<user-name>/<repo-name>/projects >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
-            "Welcome to the all-new projects": "欢迎访问全新的项目",
-            "Built like a spreadsheet, project tables give you a live canvas to filter, sort, and group issues and pull requests. Tailor them to your needs with custom fields and saved views.": "构建像电子表格一样的项目表，给您一个实时的画布来对议题和拉取请求进行筛选、排序和分组。通过自定义字段和保存的视图，使它们符合您的需要。",
-
+            "Welcome to projects": "欢迎访问项目",
+            "Built like a spreadsheet, project tables give you a live canvas to filter, sort, 和 group issues and pull requests. Tailor them to your needs with custom fields and saved views.":"项目表格就像电子表格一样，为您提供一个实时编辑的画布，用于筛选、排序和分组问题及拉取请求。您可以根据需求自定义字段并保存视图",
             "This repository doesn't have any projects yet": "该仓库目前没有任何项目",
             "Create a project": "创建一个项目",
+            "Provide quick access to relevant projects.":"为相关项目提供便捷访问",
+            "Add projects to view them here.":"添加项目以在此处查看。",
+            "Link a project":" 链接一个项目",
+            "Link projects":"链接项目",
```

Comment:
```suggestion
            "Link a project":"关联一个项目",
            "Link projects":"关联项目",
```
提醒：不少存量的`link XXX` 均可调整为`关联...`